### DISCUSSION TITLE
Items without Assets

### DISCUSSION THREAD

**Discussion by zstatmanweil** [radiantearth/stac-spec#1313] [General STAC Q&A] [answered]
I am processing data along a custom grid and I want to store metadata for every grid cell, even if no data was found, in order to document that data processing was attempted. Do other folks use STAC in this manner? Is it reasonable to have an Item without Assets? I don't see anything in the spec explicitly but I do see `rstac` uses a `has_assets` function so others must create Items without Assets. 

Thanks!

**Comment by m-mohr** [comment, marked as answer]
The schema for the assets in Items defines no minimum requirements, so the assets can be empty:
https://github.com/radiantearth/stac-spec/blob/master/item-spec/json-schema/item.json#L298

I can't find it right now, but if my memory serves me right there's somewhere a mention in the spec that "at least one asset is strongly recommended" (i.e. not required).

So I'd say it's not recommended genereally, but if it works well for your usecase then it's fine to have no assets.

**Reply by zstatmanweil** [reply]
thank you!
